projects/m1/035-the-twelve-days-of-christmas/js/index.js

```diff
@@ -0,0 +1,20 @@
+
+function showMessageSong() {
+const songVerses = [" two turtle doves", "three french hens", 'four calling (colly) birds', "five golden (gold) rings", "six geese a-laying", 'seven swans a-swimming', "eight maids a-milking", "nine ladies dancing", 'ten lords a-leaping', "eleven pipers piping", "twelve drummers drumming"];
+//this variable contain the array with 12 verses of the song.
+const newSongVerses = [];
+//this variable contain an empty array,it will contain the elements generated by the loop.
+for (let i = 0; i < songVerses.length; i++) {
+ //the for loop reads the elements in the array, each time it reads a new element it stops and starts over, printing the elements it reads each time
+  const singleVerse = songVerses[i];
+//this variable will contain the verses of the song read by the loop.
+  newSongVerses.push(singleVerse);
+ //the Push method inserts the various elements read and printed in the loop into the newly created array
+console.log(`On the twelfth day of Christmas, my true love sent to me` + ` ${newSongVerses.reverse()}` + ' and a partridge in a pear tree');
```

Comment:
It's **not a good idea** to **print inside** the function. 
Better to **return a value** and print it.
Example:

`return newSongVerses.join(" ")`

`console.log(showMessageSong())`

Reply:
I don't understand...how can i use this example?